activesupport/lib/active_support/file_evented_update_checker.rb

```diff
@@ -0,0 +1,73 @@
+require 'listen'
+
+module ActiveSupport
+	class FileEventedUpdateChecker
+		attr_reader :listener
```

Comment:
It seems like you're indentation is off. Can you switch to use 2 spaces as the indentation unit throughout? :smile:

It would look like this:

```ruby
module ActiveSupport
  class FileEventedUpdateChecker
    attr_reader :listener
    
     # The rest of your code...
  end
end
```

Remember to strip whitespace at the end of every line too (and empty lines should have no spaces) :smile: